css/chat.css

```diff
@@ -67,6 +58,10 @@ html {
   overflow: hidden;
 }
 
+.fl-with-top-menu .chat-holder {
+  top: 44px;
```

Comment:
@hcarneiro How about iPhone X?

Reply:
@tonytlwu That won't be affected because it's already contained in an element that already takes that into consideration. But I will change it if I'm wrong.